### DISCUSSION TITLE
assertExists() in Storage::disk

### DISCUSSION THREAD

**Discussion by FranciscoCaldeira** [laravel/framework#35521] [Q&A]
Hi, I use laravel 7.X,
Now I'm testing file upload, and by the link https://laravel.com/docs/7.x/http-tests#testing-file-uploads , I can't find the description anywhere in the docs of the assertExists() method.
Many thanks,

**Comment by ivan770** [comment]
https://laravel.com/api/7.x/Illuminate/Filesystem/FilesystemAdapter.html#method_assertExists
Hope this helps

**Reply by FranciscoCaldeira** [reply]
thanks
